book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+It is worth thinking about your future self as one of the project collaborators when balancing the resources used in opening and documenting an OSH project, and the resources used to develop the hardware further.
+ 
+
+In order to make your project more open, one can work on several aspects of open source, which are outlined in the following sections.
+
+
+
+
+# Why should you use or develop open source hardware?
+
+```{figure} ../../figures/why-open-hardware.*
+---
+height: 500px
+name: why-open-hardware
+alt: 'Illustration showing the benefits of using Open Science Hardware for Research and for researchers. On top, the figure shows a cycle divided in two parts, on the bottom, there is a person with short hair surrounded by a microscope, a computer and electronic elements. In its hand, it has a pile of paper lebelled “how to”, which represents the hardware documentation. The lower half circle is labelled “Science” and is surrounded with several half full bechers. Inside the half circle one can read four bullet points: Quality, Reproducibility, Inclusiveness, and Sustainability. The upper half circle is labelled “YOU”, it is surrounded by one person with goggles and earrings. Inside the half circle one can read four bullet points: Flexibility and speed, Price, Control, and Visibility., It represents the benefits of the Open Hardware practice for the individual researcher. These include greater flexibility and speed, as experimental set ups can easily be modified without depending on vendors. Another benefit is price, as open science hardware is today less expensive than available proprietary options. Control is another benefit for the individual researcher, as new features can be included, others removed and set ups completely customized beyond what is established by a third party vendor. Finally, individual researchers gain visibility when they develop their hardware work open, increasing the impact of their work'
+---
+_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+## For you
+
+People develop and share OSH for a variety of reasons. Particularly in research, OSH provides very concrete advantages:
+
+- Flexibility and Speed: you can quickly customize and combine open designs to test new research questions in an accessible way, instead of depending on vendors, their timelines and bureaucracy.
+
+- Simplicity: As OSH has a lower price tag than conventional equipment, it is much easier to obtain while bypassing the bureaucracy of contracting vendors.
+
+
+- Control: if a supplier goes out of business, users and/or third party companies have the information to keep systems running. 
+
+- Visibility: researchers and engineers developing OSH make their work more visible. They also increase opportunities for networking and impact,especially in project involving a community.
```

Comment:
```suggestion
- Visibility: researchers and engineers developing OSH make their work more visible to more people. 
                   They also increase opportunities for networking and impact, especially in projects involving a community.
```

Reply:
Slight wording tweak + updating the bullet formatting as per: https://the-turing-way.netlify.app/community-handbook/style#a-note-on-bullet-points